website/content/api-docs/system/internal-ui-default-auth-methods.mdx

```diff
@@ -0,0 +1,52 @@
+---
+layout: api
+page_title: /sys/internal/ui/default-auth-methods - HTTP API
+description: The '/sys/internal/ui/default-auth-methods' endpoint returns any configured or inherited default and backup auth methods for a given namespace.
+---
+
+# `/sys/internal/ui/default-auth-methods`
+
+@include 'alerts/enterprise-only.mdx'
+
+The `/sys/internal/ui/default-auth-methods` endpoint is used to return explicitly set or inherited default and backup auth method types for a given namespace.
+
+The namespace is derived from the request itself in the `X-Vault-Namespace` header.
+
+This should only be used internally by the UI. Due to the nature of its
+intended usage, there is no guarantee on backwards compatibility for this endpoint.
+
+## Get UI default login configuration
+
+This endpoint returns any configured or inherited default and backup auth methods for a given namespace.
```

Comment:
```suggestion
@include 'alerts/unrestricted.mdx'

Return any configured, or inherited, default and backup authentication methods
for a given namespace.
```
Style correction: avoid "this" as a pronoun, prefer full words over abbreviations